test_data/subgraph.json

```diff
@@ -1,34 +1,41 @@
 {
```

Comment:
if the Graph's node doesn't include attr, the function NewGraphJSON still works?

Reply:
the attr is initialed to be 0, so it still works(set attr to 0)